src/create.rs

```diff
@@ -449,7 +449,6 @@ mod tests {
         assert_eq!(res.unwrap().1, SqlType::Bigint(20));
         let res = type_identifier(CompleteByteSlice(type2.as_bytes()));
         assert_eq!(res.unwrap().1, SqlType::UnsignedBigint(20));
-        let res = type_identifier(CompleteByteSlice(type3.as_bytes()));
```

Comment:
Better change is to add the missing assertion here:
```rust
        assert_eq!(res.unwrap().1, SqlType::SignedBigint(20));
```

Reply:
Sorry, that should have been `SqlType::Bigint(20)`.

Reply:
Thanks for reviewing @ms705 and providing the *real* fix. I noticed it could not find the `SignedBigint` variant so I used `SqlType::Bigint`. Do you forsee any problems with this approach?